app/test/specs/eventControllerSpec.js

```diff
@@ -0,0 +1,118 @@
+                'text': 'Inaugurar o projeto E-CIS',
+                'local': 'Brasília',
+                'photo_url': null,
+                'start_time': date_now, 
+                'end_time': date_now,
+                };
+
+  var event_convert_date = new Event(event, splab.key);
+
+  beforeEach(module('app'));
+
+  beforeEach(inject(function($controller, $httpBackend, $http, $q, AuthService, $rootScope, ImageService, EventService, MessageService) {
+      imageService = ImageService;
+      scope = $rootScope.$new();
+      httpBackend = $httpBackend;
+      rootScope = $rootScope;
+      deffered = $q.defer();
+      eventService = EventService;
+      AuthService.login(maiana);
+      eventCtrl = $controller('EventController', {
+            scope: scope,
+            imageService : imageService,
+            $rootScope: rootScope,
+            eventService: eventService,
+            message : MessageService
+        });
+      eventCtrl.event = event;
+
+      httpBackend.when('GET', "/api/events").respond([]);
+      httpBackend.when('GET', 'main/main.html').respond(200);
+      httpBackend.when('GET', 'home/home.html').respond(200);
+      httpBackend.when('GET', 'auth/login.html').respond(200);
+      httpBackend.flush();
+  }));
+
+  afterEach(function() {
+      httpBackend.verifyNoOutstandingExpectation();
+      httpBackend.verifyNoOutstandingRequest();
+  });
+
+  describe('EventController functions', function() {
+
+    describe('showButtonSend()', function() {
+
+      it('should be true', function() {
+        expect(eventCtrl.showButtonSend()).toBe(true);
+      });
+
+      it('should be false', function() {
+        eventCtrl.event = new Event({
+              'text': 'Inaugurar o projeto E-CIS',
+                         'photo_url': null,
+                        'start_time': new Date(), 
+                         'end_time': new Date(),
+                         'key': '12300'
+                },
+              splab.key);
+        expect(eventCtrl.showButtonSend()).toBe(false);
+      });
+
+      it('should be false', function() {
+        eventCtrl.event = new Event({
+                        'title': 'Inauguração',
+                        'photo_url': null,
+                        'start_time': new Date("October 13, 2014 11:13:00"), 
+                        'end_time': new Date("October 3, 2014 11:13:00"),
+                        'key': '12300'
+                       },
+                       splab.key);
+        expect(eventCtrl.showButtonSend()).toBe(false);
+      });
+    });
+
+    describe('showImage()', function() {
+
+      it('should be false', function() {
+        expect(eventCtrl.showImage()).toBe(false);
+      });
+
+      it('should be true', function() {
+        eventCtrl.photoUrl = 'image';
+        expect(eventCtrl.showImage()).toBe(true);
+       });
+      });
+
+    describe('save()', function() {
+
+      it('should eventService.createEvent was called', function() {
```

Comment:
"was" não faz sentido, "save() should call ...alguma coisa..." ou "should ... alguma coisa.. be called"